images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -284,3 +284,18 @@ function Get-PipxVersion {
     $pipxVersion = $Matches.Version
     return "Pipx $pipxVersion"
 }
+
+function Get-GraalVMVersion {
+    $version = & "$env:GRAALVM_11_BIN\java" --version | Out-String | Select-Object -First 2 | Take-OutputPart -Part 7
```

Comment:
Why do we need `Select-Object -First 2` here?

Reply:
To select the necessary string of output

Reply:
`Out-String` transforms all output to the single line.
Then `Select-Object -First 2` tries to take 2 first lines from array with one line.